drakcore/drakcore/postprocess/wireshark_key_file_gen.py

```diff
@@ -0,0 +1,39 @@
+import json
+import os
+import logging
+from karton2 import Task, RemoteResource
+from typing import Dict
+from tempfile import NamedTemporaryFile
+
+
+def gen_key_file_from_log(tlsmon_log):
+    key_file_content = ''
+    for line in tlsmon_log:
+        try:
+            entry = json.loads(line)
+            client_random = entry['client_random']
+            master_key = entry['master_key']
+            key_file_entry = f'CLIENT_RANDOM {client_random} {master_key}\n'
+            key_file_content += key_file_entry
+        except KeyError:
+            logging.exception(f"JSON is missing a required field\n{line}")
+            continue
+        except json.JSONDecodeError as e:
+            logging.warning(f"line cannot be parsed as JSON\n{e}")
+            continue
+    key_file = NamedTemporaryFile(delete=False)
```

Comment:
I believe that `BytesIO` would be better here (refer to pstree.py postprocessing)